lib/amplitude.dart

```diff
@@ -304,6 +304,15 @@ class Amplitude extends _Amplitude {
         'setMinTimeBetweenSessionsMillis', jsonEncode(properties));
   }
 
+  /// Android only, allows to provide a custom advertising id
+  Future<void> setCustomAdvertisingId(String advertisingId) async {
```

Comment:
Wonder if need some placeholder method in iOS and web @yuhao900914

Reply:
Yes, that's right. Both ios and web need a placeholder otherwise it's not able to build.  @vishna  Can you please update this?